docs/example-react-intl.md

```diff
@@ -0,0 +1,51 @@
+---
+id: example-react-intl
+title: React Intl
+---
+
+```jsx
+import React from 'react'
+import 'jest-dom/extend-expect'
+import { render, cleanup, getByTestId } from 'react-testing-library'
+import { IntlProvider, FormattedDate } from 'react-intl'
+import IntlPolyfill from 'intl'
+import 'intl/locale-data/jsonp/pt'
+
+const setupTests = () => {
```

Comment:
Can this link to the setup section that discusses setting up globals?

Reply:
Move the setup description of react-intl to the /docs/react-testing-library/setup or the /docs/setup?

Reply:
Just add a link here to `/docs/react-testing-library/setup` to clarify how this can be combined with other global setup functions

Reply:
I've done that and created a section at the setup with a small explanation about why is necessary to load locales.
Do you think it's ok?

Reply:
Can you move the description in setup to this file instead to keep it all in one place, and keep setup generic?

Reply:
Done.